akka-projection-persistence-query-provider/src/main/scala/akka/persistence/query/TypedEventEnvelope.scala

```diff
@@ -0,0 +1,25 @@
+package akka.persistence.query
```

Comment:
Choosing to keep the akka original package. Maybe something to move to Akka and some utility class to run queries returning typed envelopes.